dashboard/app/views/levels/editors/fields/_instructions.haml

```diff
@@ -2,6 +2,6 @@
   Instructions
 
 #instructions.in.collapse
-  = render partial: 'levels/editors/fields/short_instructions', locals: {f: f} unless (@level.is_a?(Applab) || @level.instance_of?(Gamelab) || @level.is_a?(Weblab) || @level.is_a?(ExternalLink) || @level.is_a?(FreeResponse) || @level.is_a?(FrequencyAnalysis) || @level.is_a?(PublicKeyCryptography)|| @level.is_a?(Vigenere) || @level.is_a?(Javalab))
+  = render partial: 'levels/editors/fields/short_instructions', locals: {f: f} unless (@level.is_a?(Applab) || @level.instance_of?(Gamelab) || @level.is_a?(Weblab) || @level.is_a?(ExternalLink) || @level.is_a?(FreeResponse) || @level.is_a?(FrequencyAnalysis) || @level.is_a?(PublicKeyCryptography)|| @level.is_a?(Vigenere) || @level.is_a?(Javalab)) || @level.is_a?(Aichat)
```

Comment:
What does adding `Aichat` to this line mean - that the AI chat's instructions does NOT toggle between short and long? Just curious.

Reply:
tbh I don't know exactly -- I followed a lot of the wisdom in this PR from Dani, who suggested that long instructions were the way:

https://github.com/code-dot-org/code-dot-org/pull/39631

Reply:
Interesting - didn't know short instructions were only meant for CSF style levels. Thanks!